lib/fog/vsphere/models/compute/server.rb

```diff
@@ -37,6 +37,7 @@ class Server < Fog::Compute::Server
         attribute :corespersocket
         attribute :interfaces
         attribute :volumes
+        attribute :cdroms
```

Comment:
You don't need this anymore since the `has_many_identities` will define it for you.